### DISCUSSION TITLE
The getter of the TypeScript is unexpected

### DISCUSSION THREAD

**Discussion by wyy-xindong** [vercel/next.js#27954] [Help]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

14.15.4

### What browser are you using?

Chrome

### What operating system are you using?

Windows10 2004

### How are you deploying your application?

npm run dev

### Describe the Bug

There is an inheritance in the code,we declare an abstract property in the parent class and  implement the getter of the property in the subclass.And then,we instantiate the subclass and call the getter we mentioned above.Now the bug appears,the getter method is unrecognized,we cannot get the returned value of the getter. Instead, we get a uninitialized property declared in the parent class,it is `undefined`.However,under the same conditions, we can get the returned value of the getter according to the TypeScript official introduction.

### Expected Behavior

The getter should be recognized,we should get the returned value of the getter in this call.

### To Reproduce

Preparations:
1.Start a Docker image by command `docker run --rm -it -p 3000:3000 node:14.15 /bin/bash`.
2.Create a TypeScript project by command `npx create-next-app --ts`.
3.Paste test code to the  file `./pages/_app.tsx` .
```ts
export abstract class Parent{
  abstract a: number
}
export class Sub extends Parent{
  private _a = 0
  get a(){
    return this._a
  }
}

function test(){
  const s = new Sub()
  const s_a = s.a
  console.log(s_a)
}
```
The s_a should be 0, but it is undefined.

**Comment by LetItRock** [comment]
hi @wyy-xindong! 
you should specifically define in the abstract class the getter like: `abstract get a(): number` then it will work. 
Or in the `Sub` class do `a = this._a` instead if you don't want to do changes in the abstract class ;)

**Reply by AnnatarHe** [reply]
I don't think the answer you provided is good enough.
I'm very confused that why the `typescript in next` can not follow the `origin typescript` specification?

**Reply by wyy-xindong** [reply]
hi @LetItRock !
Think you for your reply.Thanks to your answer, I can run my program.But I think it will be better if typescript in next can follow the origin typescript specification,too.
